app/models/wiki.rb

```diff
@@ -120,6 +120,17 @@ def course_prefix
     edit_templates['course_prefix']
   end
 
+  def string_prefix
+    case project
+      when 'wikipedia'
```

Comment:
This isn't standard Ruby formatting. Try running `bundle exec rubocop` or configure your editor to do rubocop syntax highlighting.